client/extensions/woocommerce/components/reading-widget/index.js

```diff
@@ -0,0 +1,197 @@
+		query: { category: 'start-your-store', number: 3 },
+		siteId: 113771570,
+	}
+
+	getRolesAndLabels = () => {
+		const { translate } = this.props;
+
+		return [
+			{ value: 'storeOwner', label: translate( 'Store owner' ) },
+			{ value: 'developer', label: translate( 'Developer' ) },
+			{ value: 'extensionDeveloper', label: translate( 'Extension developer' ) },
+			{ value: 'other', label: translate( 'Other' ) },
+		];
+	};
+
+	handleSubscriptionEmailFocus = () => {
+		this.setState( { subscriptionFormExpanded: true } );
+	}
+
+	renderHeading = () => {
+		const { text, title } = this.props;
+		return (
+			<div className="reading-widget__heading">
+				<h2>{ title }</h2>
+				<p>{ text }</p>
+			</div>
+		);
+	}
+
+	renderArticleList = () => {
+		const { posts, query, siteId } = this.props;
+
+		return (
+			<div>
+				<QueryPosts
+					query={ query }
+					siteId={ siteId }
+				/>
+				<ul className="reading-widget__article-list">
+					{
+						( posts && posts.map( post => {
+							const { date, ID, title, URL } = post;
+							return (
+								<li key={ ID }>
+									<ExternalLink icon={ true } href={ URL } rel="noopener noreferrer" target="_blank">
+										{ title }
+									</ExternalLink>
+									<span>{ humanDate( date ) }</span>
+								</li>
+							);
+						} ) )
+					}
+				</ul>
+			</div>
+		);
+	}
+
+	renderSubscriptionFormExplanation = ( expanded ) => {
+		const { translate } = this.props;
+
+		return (
+			<div className="reading-widget__subscription-form-explanation">
+				<p>
+					{ translate( 'Subscribe to our newsletter and get the latest' +
+						' news delivered to your inbox' ) }
+				</p>
+				{
+					expanded && ( <p>
+						{ translate( 'Before you subscribe, please specify your role(s)' +
+							' in eCommerce and we\'ll send you the most relevant news & info' ) }
+					</p> )
+				}
+			</div>
+		);
+	}
+
+	onEmailFocus = () => {
+		this.setState( {
+			subscriptionFormExpanded: true,
+		} );
+	}
+
+	onEmailChange = ( e ) => {
+		this.setState( {
+			emailAddress: e.target.value
+		} );
+	}
+
+	onRolesChange = ( e ) => {
+		this.setState( {
+			roles: e.value
+		} );
+	}
+
+	isFormSubmittable = () => {
+		const emailValidates = emailValidator.validate( this.state.emailAddress );
+		const roleSelected = ( 0 < this.state.roles.length );
+
+		return emailValidates && roleSelected;
+	}
+
+	onSubmit = () => {
+		// TODO
+	}
+
+	renderSubscriptionFormFields = ( expanded ) => {
+		const { translate } = this.props;
+		return (
+			<div className="reading-widget__subscription-form-fields">
+				<FormTextInput
+					onChange={ this.onEmailChange }
+					onFocus={ this.onEmailFocus }
+					placeholder={ translate( 'hello@email.com' ) }
+					value={ this.state.emailAddress }
+				/>
+				{ expanded && (
+					<MultiCheckbox
+						checked={ this.state.roles }
+						onChange={ this.onRolesChange }
+						options={ this.getRolesAndLabels() }
+					/>
+				) }
+				{ expanded && (
+					<Button
```

Comment:
Agreed! Fixed in https://github.com/Automattic/wp-calypso/pull/14339/commits/fc929a9089c59738059bec4d882378d93712696e